style.css

```diff
@@ -329,8 +461,82 @@ body.lock {
         display: none;
     }
     .content {
-        padding: 50px 0 0;
+        padding: 0 0;
     } 
+    .carousel__text {
+        top: 10%;
+        left: 3%;
+    }
+    .carousel__text1 {
+        color: var(--element-color);
+        font-size: 28px;
+        letter-spacing: 3px;
+    }
+    .carousel__text2 {
+        color: var(--additional-color);
+        font-size: 25px;
+        letter-spacing: 1.5px;
```

Comment:
Can you pelase explain how you achieve 1.5px on a monitor?

Reply:
Retina display?

Reply:
Retina has double pixel density, they are not able to display half of a pixel it is physically impossible. All you are making here for yourself is inconsistency issue between different monitors. 

https://en.wikipedia.org/wiki/Subpixel_rendering

Reply:
"Pixels in CSS are "device independent" or "relative" pixels. They're an abstract unit of measure. On high DPI/PPI devices, a single CSS pixel may become two physical pixels. A 4px measure may become 6px, or 8px. Fractional pixels are actually quite useful as a 0.5px border should round up to 1px at 1.0 scale factor, and remain 1px (as thin as possible) at 1.5 or 2.0 scale"  from here https://stackoverflow.com/questions/20116132/half-pixel-in-border-width-size-it-is-not-showing